src/js/state/rootReducer.ts

```diff
@@ -20,8 +20,10 @@ import Toolbars from "./Toolbars"
 import PluginStorage from "./PluginStorage"
 import Configs from "./Configs"
 import ConfigPropValues from "./ConfigPropValues"
+import Launches from "./Launches"
 
 export default combineReducers<any, any>({
+  launches: Launches.reducer,
```

Comment:
I think you'll also want to add this to the global reducer, no?

Reply:
Yes I do!